src/examples/java/com/amazonaws/crypto/examples/datakeycaching/MultiRegionRecordPusherExample.java

```diff
@@ -73,10 +72,10 @@ public MultiRegionRecordPusherExample(final Region[] regions, final String kmsAl
                     .build());
 
             keyrings.add(StandardKeyrings.awsKmsBuilder()
-                    .awsKmsClientSupplier(AwsKmsClientSupplier.builder()
+                    .awsKmsClientSupplier(new AllowRegionsAwsKmsClientSupplier(Collections.singleton(region.getName()),
```

Comment:
You have all of these nice builders, why is this one `new`?

Reply:
Seemed like it might be overkill to have a builder here, but I can make one too. I wasn't really satisfied with the mix of builders and constructors either